src/check_lines.rs

```diff
@@ -190,6 +190,25 @@ pub fn check_lines(
         }
     }
 
+    // Enfore newlines at end of files. Don't check if noqa enabled.
+    if settings.enabled.contains(&CheckCode::W292) && !enforce_noqa {
+        // If the file contains a newline at the end of it the last
+        // line would contain an empty string slice.
+        let line = lines.last().unwrap_or(&"");
```

Comment:
Alternatively, you could do:

```rust
if let Some(line) = lines.last() {
  ...
}
```

(Since we're never gonna trigger the rule if we have no lines.)

Reply:
It's funny. This is what I had originally. Will do.